scripts/main/view.js

```diff
@@ -25,96 +32,92 @@ view.albums = {
-					smartData += build.album(albums.json.smart_albums.public);
-				}
-				if (albums.json.smart_albums.starred) {
-					albums.parse(albums.json.smart_albums.starred);
-					smartData += build.album(albums.json.smart_albums.starred);
-				}
-				if (albums.json.smart_albums.recent) {
-					albums.parse(albums.json.smart_albums.recent);
-					smartData += build.album(albums.json.smart_albums.recent);
-				}
-
-				Object.entries(albums.json.smart_albums).forEach(([, albumData]) => {
-					if (albumData["is_tag_album"]) {
-						albums.parse(albumData);
-						smartData += build.album(albumData);
-					}
-				});
+			if (lychee.publicMode === false) {
+				smartData = build.divider(lychee.locale["SMART_ALBUMS"]);
+			}
+			if (albums.json.smart_albums.unsorted) {
+				albums.parse(albums.json.smart_albums.unsorted);
+				smartData += build.album(albums.json.smart_albums.unsorted);
+			}
+			if (albums.json.smart_albums.public) {
+				albums.parse(albums.json.smart_albums.public);
+				smartData += build.album(albums.json.smart_albums.public);
+			}
+			if (albums.json.smart_albums.starred) {
+				albums.parse(albums.json.smart_albums.starred);
+				smartData += build.album(albums.json.smart_albums.starred);
+			}
+			if (albums.json.smart_albums.recent) {
+				albums.parse(albums.json.smart_albums.recent);
+				smartData += build.album(albums.json.smart_albums.recent);
 			}
 
-			// Albums
-			if (albums.json.albums && albums.json.albums.length !== 0) {
-				$.each(albums.json.albums, function () {
-					if (!this.parent_id) {
-						albums.parse(this);
-						albumsData += build.album(this);
-					}
-				});
+			// Tag albums
+			tagAlbumsData += albums.json.tag_albums.reduce(function (html, tagAlbum) {
+				albums.parse(tagAlbum);
+				return html + build.album(tagAlbum);
+			}, "");
 
-				// Add divider
-				if (lychee.publicMode === false) albumsData = build.divider(lychee.locale["ALBUMS"]) + albumsData;
-			}
+			// Albums
+			if (lychee.publicMode === false) albumsData = build.divider(lychee.locale["ALBUMS"]);
+			albumsData += albums.json.albums.reduce(function (html, album) {
+				albums.parse(album);
+				return html + build.album(album);
+			}, "");
 
 			let current_owner = "";
-			let i;
 			// Shared
-			if (albums.json.shared_albums && albums.json.shared_albums.length !== 0) {
-				for (i = 0; i < albums.json.shared_albums.length; ++i) {
-					let alb = albums.json.shared_albums[i];
-					if (!alb.parent_id) {
-						albums.parse(alb);
-						if (current_owner !== alb.owner_name && lychee.publicMode === false) {
-							sharedData += build.divider(alb.owner_name);
-							current_owner = alb.owner_name;
-						}
-						sharedData += build.album(alb, !lychee.admin);
-					}
+			sharedData += albums.json.shared_albums.reduce(function (html, album) {
+				albums.parse(album);
+				if (current_owner !== album.owner_name && lychee.publicMode === false) {
+					sharedData += build.divider(album.owner_name);
```

Comment:
```suggestion
					html += build.divider(album.owner_name);
```

Reply:
I cannot accept this suggestions anymore, because it refers to an outdated version of the file. But unlike the other suggestions it seems not to be fixed yet.

Reply:
Checked the actual code base. It has already been fixed, so I am going to resolve this conversation.

Reply:
Sorry; I should've deleted these suggestions after including them in other commits...